react/features/base/react/components/native/NavigateSectionList.js

```diff
@@ -274,6 +279,34 @@ export default class NavigateSectionList extends Component<Props> {
         return null;
     }
 
+    _renderListEmptyComponent: () => Object
+
+    /**
+     * Renders a component to display when the list is empty.
+     *
+     * @private
+     * @param {Object} section - The section being rendered.
+     * @returns {React$Node}
+     */
+    _renderListEmptyComponent() {
+        const { onRefresh } = this.props;
+
+        if (typeof onRefresh === 'function') {
```

Comment:
Is there a time when it is undefined? I'm leaning towards preferring the style of marking a prop as required and forcing the prop to be defined. If onRefresh isn't defined, is it proper UX to return null for the React Node? (I'm asking out of ignorance.)

Reply:
onRefresh can be undefined, when the section list cannot be refreshed, e.g. in case of RecentList.

What do you mean by return null for the React Node?